src/Servers/Connections.Abstractions/src/Features/IStreamListener.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Connections.Features
+{
+    public interface IStreamListener
```

Comment:
I wonder if these should be Quic specific features?

Reply:
I think they should for now. Rename to IQuicStreamListener?